lib/travis/api/v3/models/cron.rb

```diff
@@ -13,7 +13,7 @@ def next_enqueuing
       elsif last_cron_build_date >= planned_time(LastBuild)
         planned_time(ThisBuild)
       else
-        Time.now
+        Time.now - 5.minutes
```

Comment:
@BanzaiMan Crons aren't being picked up currently. I'm trying to figure out why. There is an if statement that determines if a cron should be run up the stack from here when scheduler calls it. I was concerned that the Time.now in that if was evaluated before this one by enough of a time difference to prevent them from running (I was seeing data to support this and I'm currently testing it on staging). However, there could be other causes and this may not be necessary.